src/accelerate/utils/dataclasses.py

```diff
@@ -1312,3 +1320,132 @@ def set_tensorboard_logging_options(self):
                 self.megatron_lm_default_args[key] = True
             elif key.startswith("no_log_"):
                 self.megatron_lm_default_args[key.replace("no_", "")] = True
+
+
+@dataclass
+class BnbQuantizationConfig:
+    """
+    A plugin to enable BitsAndBytes 4bit and 8bit quantization
+    """
+
+    load_in_8bit: bool = field(default=False, metadata={"help": "enable 8bit quantization."})
+
+    llm_int8_threshold: float = field(default=6.0, metadata={"help": "value of the outliner threshold."})
```

Comment:
Maybe specify that it is relevant only for `load_in_8bit=True` (the arg name is explicit but we never know :D )

Reply:
done =)